doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -19,23 +22,66 @@ version to install - when not specified, the latest version will be installed in
 
 ## Upgrading the Operator
 
-To upgrade the Operator, use the dedicated `upgrade-operator` command.
+The procedure for upgrading the Operator depends on the version of the Operator you want to
+upgrade from.
+
+### Upgrading the Operator (>v1.0)
+
+Astarte Operator's Helm chart is available since v1.0 and therefore the current upgrade procedure applies only
+if you are dealing with at least v1.0.
+
+To upgrade the Operator, use the dedicated `helm upgrade` command:
 
 ```bash
-astartectl cluster upgrade-operator
+helm upgrade astarte-operator astarte/astarte-operator [--version <new-version>]
 ```
 
-Just like the `install-operator` command, the `--version` switch allows to specify the version to upgrade to -
-when not specified, the latest version will be fetched and used.
+The optional `--version` switch allows to specify the version to upgrade to - when not specified, the latest
+version will be fetched and used.
+
+Currently, only the first version of the chart is available within `helm.astarte-platform.org` and the `helm upgrade`
+operation will lead to an error. Bear with that as long as new versions will be released.
+
+### Upgrading the Operator (from v0.11 to v1.0)
+
+The upgrade procedure from v0.11 to v1.0 may be thought of as non-linear procedure as the deployment and handling of
```

Comment:
"may be thought of as non-linear procedure" -> "requires some manual intervention"